conans/client/settings_preprocessor.py

```diff
@@ -0,0 +1,18 @@
+from conans.errors import InvalidConanSettingField
+
+
+def preprocess(settings, output):
+    fill_runtime(settings, output)
+
+
+def fill_runtime(settings, output):
+    try:
+        if settings.compiler == "Visual Studio":
+            if settings.get_safe("compiler.runtime") is None:
+                settings.compiler.runtime = "MDd" if settings.get_safe("build_type") == "Debug" \
```

Comment:
Then it has to be opt-out at least. We cannot force the user to specify MD => Release, MDd => Debug.
I would say that we shouldn't make that check.